c2corg_ui/attributes.py

```diff
@@ -43,3 +43,21 @@
     'paragliding_takeoff',
     'paragliding_landing'
 ]
+
+# FIXME: incorrect values
```

Comment:
Do you want to use the file from the common project?

These are the change I made to the api project:
https://github.com/c2corg/v6_api/commit/ff5b61cfb9b8c55c109fdd294418afe8f85b887b